cloud/validations.go

```diff
@@ -104,7 +104,7 @@ func validateCloud(data []byte, jsonSchema *map[string]interface{}) error {
 	formatKeyError := func(invalidKey, similarValidKey string) string {
 		str := fmt.Sprintf("property %s is invalid.", invalidKey)
```

Comment:
I'm also a little unsure about ". Perhaps". Given that we prefer to write errors in lowercase ("property ... is invalid"), it feels weird to start with lowercase and then create a full stop and new sentence in the middle.
Maybe ; or , would be more appropriate and then it would read:

property "foo" is invalid; perhaps you mean "foob"

And then avoid the full stops in errors completely.